cli/src/main/java/com/expediagroup/streamplatform/streamregistry/cli/action/GraphQLEventSenderAction.java

```diff
@@ -22,16 +22,23 @@
 import picocli.CommandLine.Option;
 
 import com.expediagroup.streamplatform.streamregistry.state.EventSender;
+import com.expediagroup.streamplatform.streamregistry.state.graphql.Credentials;
 import com.expediagroup.streamplatform.streamregistry.state.graphql.DefaultApolloClientFactory;
 import com.expediagroup.streamplatform.streamregistry.state.graphql.GraphQLEventSender;
 
 public abstract class GraphQLEventSenderAction implements EventSenderAction {
   @Option(names = "--streamRegistryUrl", required = true)
   @Getter String streamRegistryUrl;
 
+  @Option(names = "--streamRegistryUsername", required = false)
+  @Getter String streamRegistryUsername;
+
+  @Option(names = "--streamRegistryPassword", required = false)
+  @Getter String streamRegistryPassword;
+
   @Override
   public EventSender sender() {
-    ApolloClient client = new DefaultApolloClientFactory(streamRegistryUrl).create();
+    ApolloClient client = new DefaultApolloClientFactory(streamRegistryUrl, new Credentials(streamRegistryUsername, streamRegistryPassword)).create();
```

Comment:
If they're now optional, you should only provide a credentials object if they've been provided.

Reply:
done